### PR TITLE
Use bundler icon for files with .gemfile extension

### PR DESCRIPTION
**Pull request by artplan1**
**Changes proposed:**

- [x] Add

---

Useful for files with names like `spree_master.gemfile` / `spree_master.gemfile.lock`

### CODE REVIEW

File: src/icon-manifest/supportedExtensions.ts [hunk 1]
```diff
@@ -91,6 +91,7 @@ export const extensions: IFileCollection = {
     { icon: 'bolt', extensions: [], languages: [languages.bolt], filename: true, format: FileFormat.svg },
     { icon: 'bower', extensions: ['.bowerrc', 'bower.json'], filename: true, format: FileFormat.svg },
     { icon: 'buckbuild', extensions: ['.buckconfig'], filename: true, format: FileFormat.svg },
+    { icon: 'bundler', extensions: ['gemfile', 'gemfile.lock'], format: FileFormat.svg },
```
Comment (jens1o) [inline src/icon-manifest/supportedExtensions.ts]: it is already defined a line below?

Comment (artplan1) [reply src/icon-manifest/supportedExtensions.ts]: it's defined below with `filename: true` and works only for files `gemfile` and `gemfile.lock`

**Comment by JimiC** [comment]
My only question is how common is the use of such files. Any docs?

**Comment by artplan1** [comment]
@JimiC 
for example gem `appraisal` generates files with this extensions:
https://github.com/thoughtbot/appraisal#under-the-hood

**Pull request merged**